### PR TITLE
fix-next: drawer plugin viewcontroller init

### PR DESCRIPTION
**Pull request by MartoYankov**
Fixes the RadSideDrawer appearing black in `next`. The problem was caused by https://github.com/NativeScript/NativeScript/pull/6102 . The issue was that we waited for the native views to be instantiated before we create the wrapper UILayoutViewController. RadSideDrawer checks its modes when its native view is created. It requires the view controller to be created before the native view.

### CODE REVIEW

**Comment by MartoYankov** [comment]
test pro_templates

File: tns-core-modules/application/application.ios.ts [hunk 1]
```diff
@@ -339,13 +341,21 @@ function getViewController(view: View): UIViewController {
...
-        }
+        // We set UILayoutViewController dynamically to the root view if it doesn't have a view controller
+        // At the moment the root view doesn't have its native view created. We set it in the setViewControllerView func
+        viewController = iosView.UILayoutViewController.initWithOwner(new WeakRef(view)) as UIViewController;
+        view.viewController = viewController;
+        return viewController;
+    }
+}
+
+function setViewControllerView(view: View): void {
```
Comment (manoldonev) [inline tns-core-modules/application/application.ios.ts]: Not sure what is the idea of the `setViewControllerView` naming, can you elaborate a bit?

File: tns-core-modules/application/application.ios.ts [hunk 2]
```diff
@@ -339,13 +341,21 @@ function getViewController(view: View): UIViewController {
...
+    const viewController: UIViewController = view.viewController || view.ios;
+    const nativeView = view.ios || view.nativeViewProtected;
+
+    if (nativeView && viewController && viewController instanceof iosView.UILayoutViewController) {
+        viewController.view.addSubview(nativeView);
+        return;
     }
 
     throw new Error("Root should be either UIViewController or UIView");
```
Comment (manoldonev) [inline tns-core-modules/application/application.ios.ts]: Looks like we have a scenario where viewController can be UIViewController but not UILayoutViewController -- previously we did not throw exception in this case but with the current flow I think we will.

File: tns-core-modules/application/application.ios.ts (hunk 1 above)
Comment (MartoYankov) [reply tns-core-modules/application/application.ios.ts]: We add a subview to the view of the view controller. `viewController.view = ...`. The correct `setViewControllerViewSubView` didn't sound good to me.

**Comment by MartoYankov** [comment]
test pro_templates

**Pull request merged**